src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -88,6 +91,18 @@ def get(self, request: Request, project) -> Response:
                     and not has_latest_adopted_release
                 ):
                     continue
+                if (
+                    context["id"] == "sentry.rules.filters.tagged_event.TaggedEventFilter"
+                    or context["id"] == "sentry.rules.filters.event_attribute.EventAttributeFilter"
```

Comment:
future todo...

i feel like it'd be really cool if we changed this to be a registry pattern. that way we can just have each rule implement anything it wants to in this layer, if we created a registry and registered different life cycles of this processing, then we could move all the code that's specific to a condition into that specific condition.